src/commands/schemas.ts

```diff
@@ -8,9 +8,25 @@ import { SchemaRegistry } from "../models/schemaRegistry";
 import { KafkaTopic } from "../models/topic";
 import { ResourceManager } from "../storage/resourceManager";
 import { getSchemasViewProvider } from "../viewProviders/schemas";
+import { uploadNewSchema } from "./schemaUpload";
 
 const logger = new Logger("commands.schemas");
 
+export function registerSchemaCommands(): vscode.Disposable[] {
```

Comment:
Move this to the top of the file, plust then add in confluent.schemas.upload registration.

Reply:
We'll probably want to do this kind of reorganizing/cleanup for the other files in `src/commands/*` in a future branch, yeah?